astropy/cosmology/flrw/tests/test_flrw.py

```diff
@@ -0,0 +1,108 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+
+"""Testing :mod:`astropy.cosmology.FLRW`."""
+
+from typing import final
+
+import pytest
+
+from astropy.cosmology import FLRW
+from astropy.cosmology.core import _COSMOLOGY_CLASSES, dataclass_decorator
+from astropy.cosmology.tests.helper import get_redshift_methods
+from astropy.cosmology.tests.test_core import invalid_zs
+from astropy.utils.compat.optional_deps import HAS_SCIPY
+
+from .test_base import FLRWTest
+
+
+@dataclass_decorator
+class SubFLRW(FLRW):
+    def w(self, z):
+        return super().w(z)
+
+
+@final
```

Comment:
This (type-check-only) decorator is an addition

Reply:
Like sugar: delicious 😋